src/application-constants.ts

```diff
@@ -1,7 +1,8 @@
-export const BRIDGE_CLI_DEFAULT_PATH_MAC = '/bridge-cli' //Path will be in home
-export const BRIDGE_CLI_DEFAULT_PATH_WINDOWS = '\\bridge-cli'
-export const BRIDGE_CLI_DEFAULT_PATH_LINUX = '/bridge-cli'
-export const BRIDGE_CLI_ARTIFACTORY_URL = 'https://repo.blackduck.com/artifactory/bds-integrations-release/com/blackduck/integration/bridge-cli/'
+export const BRIDGE_CLI_DEFAULT_PATH_MAC = '/bridge-cli-bundle' //Path will be in home
+export const BRIDGE_CLI_DEFAULT_PATH_WINDOWS = '\\bridge-cli-bundle'
+export const BRIDGE_CLI_DEFAULT_PATH_LINUX = '/bridge-cli-bundle'
+// export const BRIDGE_CLI_ARTIFACTORY_URL = 'https://repo.blackduck.com/artifactory/bds-integrations-release/com/blackduck/integration/bridge-cli/'
+export const BRIDGE_CLI_ARTIFACTORY_URL = 'https://artifactory.internal.synopsys.com/artifactory/clops-local/clops.sig.synopsys.com/bridge/binaries/bridge-cli-bundle/'
```

Comment:
Why are we using an internal URL in the public repo? If you want to use it, you can pass using parameter.

Reply:
Because nothing is present in external URL and it will download bridge everytime if it does not find